happi/client.py

```diff
@@ -831,8 +876,8 @@ def from_config(cls, cfg=None):
     @staticmethod
     def _get_backend_from_config(
         cfg_section: configparser.SectionProxy,
-        cfg: str
-    ) -> _Backend:
+        cfg: Union[str, os.PathLike]
+    ) -> Optional[_Backend]:
```

Comment:
Comment pre-dating PR: This is correct but I suspect this isn't a good interface. Other code needs to both expect that this may return None or that it may raise.

Reply:
I guess this is a matter of how severe a backend instantiation failure is to us.  I think it's sensible in this case to raise if we fail to instantiate one, since it's necessary to the operation of the client.